src/modules/Dashboard/features/Applet/AddUser/InvitationsTable/InvitationsTable.utils.test.ts

```diff
@@ -1,6 +1,8 @@
 import { mockI18Next } from 'shared/tests';
 
 import { getHeadCells, getInvitationsTableRows } from './InvitationsTable.utils';
+import { Invitations } from '../AddUser.types';
+import { mockedAppletId } from '../../../../../../shared/mock';
```

Comment:
Small suggestion, though linter will complain about import order

```suggestion
import { mockedAppletId } from 'shared/mock';
```

Reply:
Thanks. Fixed